css/style.css

```diff
@@ -0,0 +1,19 @@
+
+  @media screen and (max-width: 600px) {
+    body {
+      background-color: olive;
+      height: 100vh;
+    }
```

Comment:
Indentation can be two spaces for readability. You may want to avoid using tab or change tab settings to two spaces if you have a flexible editor.

Reply:
- [ ] Always make sure to include an explanation, a "why", for each requested change, and use polite words. Kindly Implement it.

Reply:
Thank you let me implement it as below:

Perhaps for indentation, it would be ideal to use two spaces instead of tab. This would ensure the readability of your code. Kindly consider it.